src/comments.js

```diff
@@ -144,7 +144,8 @@ function attach(comments, ast, text) {
       if (
         handleMemberExpressionComment(enclosingNode, followingNode, comment) ||
           handleIfStatementComments(enclosingNode, followingNode, comment) ||
-          handleTryStatementComments(enclosingNode, followingNode, comment)
+          handleTryStatementComments(enclosingNode, followingNode, comment, true) ||
+          handleTemplateLiteralComment(enclosingNode, comment, true)
```

Comment:
Looks like I'm not consistent in adding a s or not at the end of `Comment`, we should normalize this :)